config/vufind/config.ini

```diff
@@ -840,6 +847,14 @@ authors         = Wikipedia
 url = "https://api.vlb.de/api/v1/cover/"
 ; token = "XXXXXX-XXXX-XXXXX-XXXXXXXXXXXX"
 
+; This section is needed for LocalFile cover loading.
+; Note:  All coverimage LocalFile:ConfigSection must include the following keys: 
+;        filepath, filename, and imagetype.
+[LocalFile]
+filepath = /usr/local/vufind/local/coverimages
+filename = recordid
```

Comment:
I think we should make the filename definition more flexible. For example, perhaps it should be a string with tokens in it, so you could do something like "%size%/%isbn%". The other advantage is that using a token-based system might allow us to extend this with some more sophisticated operations in the future... for example, I could imagine some substring operations might be helpful in scenarios where the files are split among directories to declutter the file system -- e.g. "123/456.jpg" instead of "123456.jpg" might be accessed using something like "%recordid:substr:0:3%/%recordid:substr:3%". Obviously we don't need to implement that now -- but it's nice to leave the door open to making it a future enhancement.